MSAL/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>1C8F.1</string>
+				<string>CA92.1</string>
+			</array>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
+		</dict>
+	</array>
+	<key>NSPrivacyCollectedDataTypes</key>
```

Comment:
I think we didn't miss anything here, but @spetrescu84 could you please double check?

Reply:
@nilo-ms, @spetrescu84, can you please check before end of this week?

Reply:
The only one I see is maybe `NSPrivacyCollectedDataTypeOtherDataTypes` for the Telemetry? We collect API results and so on in the `x-client-last-telemetry` and `x-client-current-telemetry` headers?
Or all of that falls under `NSPrivacyCollectedDataTypeOtherDiagnosticData` ?

Reply:
To add to this, we haven't added anything in Native Auth that collects any other data that's not already collected in MSAL Objective C. We do have new endpoints and such in the telemetry for example, but don't have new fields or own telemetry

Reply:
Ok, sounds good